10308.xml

```diff
@@ -0,0 +1,122 @@
+        <Operations>RW</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration>
+        </RangeEnumeration>
+        <Units>
+        </Units>
+        <Description><![CDATA[The MSISDN resource will indicate the MSISDN as assigned, if any, to the SIM card. In the case that no MSISDN is assigned to the SIM card the response should be NULL.]]></Description>
+      </Item>
+      <Item ID="4">
+        <Name>APN Retries</Name>
+        <Operations>RW</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration>
+        </RangeEnumeration>
+        <Units>
+        </Units>
+        <Description><![CDATA[The APN Retry multi instance resource will maintain the number of retries the device can attempt to activate a context on a particular APN. This resource should be defaulted to 2 retries. Should a device integrator wish to manipulate this the integrator should be able to do so.]]></Description>
+      </Item>
+      <Item ID="5">
+        <Name>APN Retry Period</Name>
+        <Operations>RW</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration>
+        </RangeEnumeration>
+        <Units>seconds</Units>
+        <Description><![CDATA[The APN retry period multi instance resource is the time in seconds between attempting a context activation on the APN. This should be defaulted to 0. Note that the IoT Module will be limited by the number retries and the periodicity of the retries by the DAM requirements in Chapter 28 of the AT&T 13340. The resource is multi-instance to accommodate different periods of retries on different APNs.]]></Description>
+      </Item>
+      <Item ID="6">
+        <Name>APN Retry Back-Off Period</Name>
+        <Operations>RW</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration>
+        </RangeEnumeration>
+        <Units>seconds</Units>
+        <Description><![CDATA[The APN Retry Back-Off Period is the period between a burst of attempts to activate a context on a particular APN. This resource is intended to be a longer period than the APN retry period to account for limitations on the numbers of times an authentication request can be made with the network. This should be defaulted to 86400 seconds (1 Day). This resource is also a multiple instance resource to accommodate different retry periods amongst different APNs.]]></Description>
+      </Item>
+      <Item ID="7">
+        <Name>SINR</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration>

```

Comment:
Here a range would be useful, and/or more details on what is the formula used / expected outcome